### ISSUE TITLE
remote audio for virtual machine

### ISSUE THREAD

**Issue by wujen** [naudio/NAudio#361]
I am using WasapiOut for a project and have no problems playing data out to devices locally.
When the application is moved to a virtualised environment (VMWare) there is a "Remote Audio" device but I get no sound when this is selected.
Playing a wav file with VLC works fine, but my WasapiOut is silent.

Am I missing something?

**Comment by markheath** [comment]
Yes, unfortunately the virtual sound card drivers in many virtualized environments / remote desktop environments don't work too well. Are you able to try WaveOutEvent instead in this scenario?
You might also try both 44.1kHz and 48kHz sample rates.

**Comment by wujen** [comment]
I'll look into that, thanks.
Unfortunately I don't have any flexibility in sampling rate. The data I am playing out in this case is a 440Hz tone sampled at 10KHz from a remote sensor.
